New package: Cloudfleet.CLI version 0.3.28

cloudfleet-tech opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/216116)

cloudfleet-tech committed:
New package: Cloudfleet.CLI version 0.3.28

stephengillie commented on manifests/c/Cloudfleet/CLI/0.3.28/Cloudfleet.CLI.installer.yaml:
```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Cloudfleet.CLI
+PackageVersion: 0.3.28
```
```suggestion
PackageVersion: dev
```
Please update file path to match this change.

stephengillie commented on manifests/c/Cloudfleet/CLI/0.3.28/Cloudfleet.CLI.locale.en-US.yaml:
```diff
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Cloudfleet.CLI
+PackageVersion: 0.3.28
```
```suggestion
PackageVersion: dev
```
Please update file path to match this change.

stephengillie commented on manifests/c/Cloudfleet/CLI/0.3.28/Cloudfleet.CLI.locale.en-US.yaml:
```diff
@@ -0,0 +1,12 @@
+PackageVersion: 0.3.28
+PackageLocale: en-US
+Publisher: Cloudfleet
+PackageName: CLI
```
```suggestion
PackageName: CloudFleet CLI
```

stephengillie reviewed (changes requested):
Verify:
![image](https://github.com/user-attachments/assets/bbdc27e1-ce12-4a05-9f28-c682d8286370)

Hi @cloudfleet-tech,

For `Portable` packages, the package managers creates a Registry entry, as seen in the screenshot above. The `PackageName` in the `defaultLocale` file becomes the package name, so a name that is similar to the package is recommended.

Separately, the package seems to report version `dev` - should this be used instead of the current `PackageVersion`? If not, should a `DisplayVersion` be used, to help users identify the version correctly? This suggestion is to help users best identiy the package, so please share your thoughts here.

cloudfleet-tech commented:
Thanks for the suggestions @stephengillie . We will sort these things out, send either an update to this version or send a new PR closing this.

cloudfleet-tech commented:
Hi @stephengillie , a new PR that addressed these issues are here. Thanks for the suggestions again and we would appreciate if you can review the new one once it passes the validations: https://github.com/microsoft/winget-pkgs/pull/216237
